android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -61,10 +63,12 @@ class DVCClient private constructor(
             try {
                 fetchConfig(user)
                 isInitialized.set(true)
-                withContext(Dispatchers.IO){
-                    eventSource = EventSource.Builder(Handler(fun () {}), URI(config?.sse?.url)).build()
-                    eventSource.start()
-                }
+                println("URL ${config?.sse?.url}")
+                eventSource = EventSource.Builder(Handler(fun () {}), URI(config?.sse?.url)).build()
```

Comment:
how come you're removing the dispatcher context here?

Reply:
i think that was moved in all the git shuffling but i put it back in the latest commit